### ISSUE TITLE
Updating for 1.0.0-beta.14

### ISSUE THREAD

**Issue by cabralRodrigo** [danibo86/ionic-typescript-definitions#6]
Somebody is working on the update for the new version of ionic? I'm needing this and I'm not sure if somebody is working on this, if not, I'll be happy to help :)

**Comment by danibo86** [comment]
Hi Rodrigo, happy to hear that someone else is interested in this project..;)
Unfortunately right now I don't have the time to make a fully conversion of the definitions to beta14.
By the way, I'm releasing an app for a customer with ionic beta14, so I made some patches to the old definitions to support the new version. But is still a work in progress (I finished updating only code interested by the app) and there's a lot of todos in the code..;)

See the new folder "beta14" and feel free to fork the project too!
